tests/test_offline_chat_director.py

```diff
@@ -75,18 +74,14 @@ def test_chat_with_online_content(client_offline_chat):
     response_message = response_message.split("### compiled references")[0]
 
     # Assert
-    expected_responses = ["http://www.paulgraham.com/greatwork.html"]
+    expected_responses = ["https://www.paulgraham.com/greatwork.html"]
```

Comment:
Did this work? It shouldn't. His website doesn't have a certificate setup. 
<img width="479" alt="Screenshot 2024-06-24 at 11 26 08" src="https://github.com/khoj-ai/khoj/assets/65192171/78d02664-dcd3-4a99-827b-966d48bf68a9">

Reply:
Yeah, I changed it because the online online chat director test was returning https link. Updated test to check if any of the http, https or www. variants are returned

_Side note: The offline chat director tests seem to hang on my machine, unsure why, need to investigate_